pages/guides/tag/[slug].js

```diff
@@ -0,0 +1,81 @@
+import Head from 'next/head';
+import { TagsPageLayout } from 'src/components';
+
+import client from '../../../src/lib/sanity';
+
+const Guides = ({ guides, navDocs, tag }) => {
+	//console.log(tag);
+
+	return (
+		<>
+			<Head>
+				<title>Synthetix - Guides</title>
```

Comment:
these strings should be added to locales and read from there

Reply:
can you please clarify? I've updated the title tag to read "Synthetix - Tags" but I don't think this is what you are referring to?